packages/inter-protocol/README.md

```diff
@@ -40,32 +42,35 @@ VaultFactory publishes data using StoredPublishKit which tees writes to off-chai
 
 The canonical keys (under `published`) are as follows. Non-terminal nodes could have data but don't yet. A `0` indicates the index of that child in added order. To get the actual key look it up in parent. High cardinality types get a parent key for enumeration (e.g. `vaults`.)
 - `published`
-    - `vaultFactory`
+    - `vaultFactory` - [snapshot of details](./test/vaultFactory/snapshots/test-vaultFactory.js.md)
         - `governance`
         - `metrics`
         - `manager0`
             - `metrics`
             - `governance`
             - `vaults`
               - `vault0`
-    - `auction`
+    - `auction` - [snapshot of details](./test/auction/snapshots/test-auctionContract.js.md)
         - `schedule`
         - `governance`
-        - `books`
-          - `book0`
-            - `data`
-    - `priceFeed`
-      - '<inputBrand>_<outputBrand>_price_feed'
-    - `psm`
+        - `book0`
+    - `reserve` - [snapshot of details](./test/reserve/snapshots/test-reserve.js.md)
+      - `governance`
+      - `metrics`
+    - `priceFeed` - [snapshot of details](./test/price/snapshots/test-fluxAggregator.js.md)
+      - `${inputBrand}-${outputBrand}_price_feed`
```

Comment:
this formerly had a `_` as in `ATOM_USD` when snapshotting shows it's `ATOM-USD`.